sites/docs/pages/components/dropdown.md

```diff
@@ -76,14 +77,14 @@ Selected: {inputs.category4.value}
     name=category4
     value=category_name
     title="Select a Category"
+    defaultValue="%"
```

Comment:
Why are these additions needed?

Reply:
Not needed, but my initial reaction to seeing the defaultValue section, is to see the use of it in the example.

Currently, we have other examples of how it can be used in other sections of dropdown, but the actual defaultValue example in the docs does not show the prop being used. 

I was just curious if this would be a more explicit example for new users.

OLD:
![image](https://github.com/evidence-dev/evidence/assets/69778837/220f0c09-8070-4d10-a766-24d96590c3da)

NEW:
![image](https://github.com/evidence-dev/evidence/assets/69778837/2c067209-439d-4fc6-9d42-4027c8590c38)

Reply:
@kwongz this example using the `<DropdownOption value=%>` is a workaround from before we had the multiselect dropdown, which had the "select all" button and a corresponding property.

I would consider removing the slot from the example and just use the prop